src/usuarios/usuarios-controlador.js

```diff
@@ -32,7 +32,10 @@ module.exports = {
     try {
       const id = await tokens.usaRefreshToken(req.body.refreshToken);
       req.user = { id };
-      this.login(req, res);
+      // isso não funciona
+      // this é setado para o objeto global
+      // TODO: descobrir como arrumar isso
+      await this.login(req, res);
```

Comment:
Aparentemente, isso aqui não funciona. O `this` fica setado para o objeto global. Você sabe como posso arrumar isso?